flang/lib/Semantics/resolve-directives.cpp

```diff
@@ -3050,4 +3040,4 @@ void OmpAttributeVisitor::IssueNonConformanceWarning(
   context_.Warn(common::UsageWarning::OpenMPUsage, source, "%s"_warn_en_US,
       warnStrOS.str());
 }
-} // namespace Fortran::semantics
+} // namespace Fortran::semantics
```

Comment:
I think that you lost a newline here (not sure what the icon means in the GitHub GUI)